src/Elements/Fields/Checkbox.php

```diff
@@ -27,6 +27,7 @@ public function getString()
         $option = $this->getValue();
         $checkbox = new Generator();
         $field = new Generator();
+        $this->setAttribute( 'value', '1' );
 
         if ($option == '1' || ! is_null($option) && $option == $this->getAttribute('value')) {
```

Comment:
`$this->getAttribute('value')` returns `null` here, thus always checking the box by default since the comparison becomes `false == null` if unchecked.